experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
+RUN mkdir /node 
+RUN date
+WORKDIR /node
+RUN mkdir Jmeter node_undertest node_baseline
+RUN git clone http://github.com/gareth-ellis/benchmarking 
+RUN bash /node/benchmarking/experimental/benchmarks/acmeair/setupJmeter.sh
+RUN numactl --hardware
+COPY node_undertest /node/node_undertest
+COPY node_baseline /node/node_baseline
+RUN /node/benchmarking/experimental/benchmarks/acmeair/run_acmeair_docker.sh
```

Comment:
If you decide to take this path, it may be worth splitting `run_acmeair_docker.sh` into two files, one for `CMD` and the other as a setup script that gets committed to the image. If possible, that setup script could be moved above so its dependencies can be purged.